src/request/components/request-detail-panel-execution.jsx

```diff
@@ -329,7 +518,7 @@ module.exports = React.createClass({
             //process post action commands
             var postCommands = null;
             if (afterActionResultMessage && endRequestMessage) { 
-                postCommands = <CommandList beforeExecuteCommandMessages={beforeExecuteCommandMessages} afterExecuteCommandMessages={afterExecuteCommandMessages} beginMessage={afterActionResultMessage} endMessage={endRequestMessage} isRoot={true} />            
+                postCommands = <CommandList beforeExecuteCommandMessages={beforeExecuteCommandMessages} afterExecuteCommandMessages={afterExecuteCommandMessages} mongoDBMessages={mongoDBMessages} beginMessage={afterActionResultMessage} endMessage={endRequestMessage} isRoot={true} />            
```

Comment:
Ideally, we could just pass all `data` messages in here, but this is an initial cut so its fine for now.